kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -161,6 +163,40 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
   @POST
   @Consumes(Array(MediaType.APPLICATION_JSON))
   def openBatchSession(request: BatchRequest): Batch = {
+    openBatchSessionInternal(request)
+  }
+
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON,
+      schema = new Schema(implementation = classOf[Batch]))),
+    description = "create and open a batch session with uploading resource file")
+  @POST
+  @Path("/upload")
```

Comment:
I'm a little bit concerned w/ this path since it's not only uploading the JAR but also creating the batch job. Comparing the existing API, the differences are
1) the accepted media type is multipart/form-data
2) it allows(or requires) the JAR as a parameter

So, would it be good to use the same path w/ opensession? Since they have different consume media types, the framework should correctly dispatch the request to the method.

cc @turboFei

Reply:
nice catch, the `upload` api looks does not make senses.

could you try to use `POST /sessions` and check whether it works?

Reply:
Yes, different accepting media type with @Consumes with same combination  HTTP method and path can be handled in different actions. 

Let's make it in both handlers with same path:
- `POST /batches` for `application/json` media type: accepting BatchRequest in JSON format from HTTP request body
- `POST /batches` for `multipart/form-data` media type`:  accepting two parts from body, including the `batchRequest` form data parm for BatchRequest in JSON format, and the other form part `resourceFile` of uploaded resource file.

Reply:
Applied changes as discussed above.